src/components/ui-gauge/ui-gauge.js

```diff
@@ -9,11 +9,8 @@ angular.module('ui').directive('uiGauge', [ '$timeout', '$interpolate',
             link: function(scope, element, attrs) {
                 $timeout(function() {
                     var gauge;
-                    var bgnd = $('#toolbar').css("background-color");
-                    //Detect panel background colour and make foreground text b/w depending.
-                    var colors = $('ui-card-panel').css("background-color").match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/);
-                    var level = ((colors[1]*299) + (colors[2]*587) + (colors[3]*114))/1000;
-                    var fgnd = (level >= 128) ? '#111' : '#eee';
+                    var bgnd = scope.$eval('me.item.waveoptions').waveColor['theme-custom'];
```

Comment:
Ok fair enough - waveColor is currently the same as the base colour and bgnd is only ever used to set the wave or gauge colour, but I agree and will update bgnd to use the base. Cheers